pyccl/ccl_correlation.i

```diff
@@ -39,4 +41,48 @@ void correlation_3d_vec(ccl_cosmology *cosmo,double a,
 
   ccl_correlation_3d(cosmo,a,nr,r,xi,0,NULL,status);
 }
+
+void correlation_multipole_vec(ccl_cosmology *cosmo,double a,double beta,
+			       int l,int ns,double *s,
+                               double *xis,int nxis,
+         		       int *status)
+{ 
+  assert(ns==nxis);
+
```

Comment:
Do we need all this empty lines in this boilerplat code? (follow whatever is done elsewhere)